build/teamcity-nightly-roachtest.sh

```diff
@@ -59,7 +59,9 @@ function upload_stats {
       # a `.` folder in ${stats_dir}, which we don't want.
       (cd "${artifacts}" && \
         while IFS= read -r f; do
-          gsutil cp "${f}" "gs://${bucket}/artifacts/${stats_dir}/${f}"
+          if [[ "${f}" != "" ]]; then
```

Comment:
Bash comparisons are a bit different than other languages. This should do the trick.

```suggestion
          if [[ -n "${f}" ]]; then
```